pages/api/address.ts

```diff
@@ -0,0 +1,19 @@
+import { NextApiRequest, NextApiResponse } from 'next'
```

Comment:
As this is not a Next.js project, we should not use those packages. Please use the appropriate packages you can find e.g. here https://vercel.com/docs/concepts/functions/serverless-functions/supported-languages#using-typescript